lib/elements/elementHelper.js

```diff
@@ -4,9 +4,12 @@ const overlayHandler = require('../handlers/overlayHandler');
 const { wait } = require('../helper');
 
 const highlightElement = async (element) => {
+  // Adding a wait as tests fail on newer versions of chrome
   if (defaultConfig.highlightOnAction.toLowerCase() !== 'true') {
-    return;
+    await wait(1000);
```

Comment:
Adding an arbitrary wait of 1s before every action would drastically increase the test execution time, I feel this is hiding some other issue behind it. I would like to investigate a little bit on this

Reply:
Sure, but please note the wait would've been there by default as highlight on action is true when running on headful mode.

Reply:
ya we have a wait in headful mode because the highlight on the element of action has to be visible for user. We should not be really needing that in headless mode, hope we should be able to find the issue that is needing this wait.

Reply:
Absolutely another note. This code block is only called in headful mode

```
    if (defaultConfig.headful) {
      await highlightElement(element);
    }
```

So the wait is not there on headless mode. This is only there on headful mode.

Reply:
oops my bad.. some how I was thinking the wait is applicable to headless mode too here if only for headful mode I feel this should be fine